WooCommerce/Classes/ViewRelated/Products/Add Product/AddProductCoordinator.swift

```diff
@@ -393,22 +393,27 @@ private extension AddProductCoordinator {
         navigationController.present(UINavigationController(rootViewController: viewController), animated: true)
     }
 
-    func buildBottomSheetPresenter(height: CGFloat? = nil) -> BottomSheetPresenter {
-        BottomSheetPresenter(configure: { bottomSheet in
+    func buildBottomSheetPresenter() -> BottomSheetPresenter {
+        BottomSheetPresenter(configure: { [weak self] bottomSheet in
+            guard let self else { return }
             var sheet = bottomSheet
             sheet.prefersEdgeAttachedInCompactHeight = true
             sheet.largestUndimmedDetentIdentifier = .none
             sheet.prefersGrabberVisible = true
```

Comment:
❓ Would it make sense to hide the grabber when on iPad? 

<img src="https://github.com/woocommerce/woocommerce-ios/assets/524475/9e6692d0-ceeb-4d50-a838-a017f00591e5" width="500"/>

Reply:
Makes sense! Removed in 9a707551192e0ddc11f6f502501501efd2082580.